### DISCUSSION TITLE
Automations are not getting triggered when camera starts recording

### DISCUSSION THREAD

**Discussion by ade-x1** [fuatakgun/eufy_security#155] [Q&A] [answered]
Hi everyone, 
I have installed Eufy_security, my camera (Eufy Pan&tilt) is recognised properly and I can can see my camera preview in the Overview page of HA. I don't really care of viewing the streaming from there, my main goal is to setup an automation that performs few tasks as soon as the camera recognise the motions. I tried to use “Camera triggers” and other triggers (like sounds) but the automation never really start. Any suggestion on what to look for?

**Comment by fuatakgun** [comment]
Hey, please use respective motion sensors rather than camera triggers. And for state, use True or False

**Reply by ade-x1** [reply]
> Hey, please use respective motion sensors rather than camera triggers. And for state, use True or False

Are those the settings you are referring to? 
<img width="1028" alt="Screenshot 2021-12-10 at 19 04 20" src="https://user-images.githubusercontent.com/59267060/145628136-c850b6bf-d888-4200-8fc9-f8b0b6ad29f7.png">

**Reply by fuatakgun** [reply]
Yes, but for state names, you should use True or False

**Reply by fuatakgun** [reply]
Motion detection enabled is about to check if motion detection function is enabled for camera.

**Reply by ade-x1** [reply]
> Motion detection enabled is about to check if motion detection function is enabled for camera.

I don't think I am doing this right, am I? I have changed the entity to binary_sensor.cerberus_motion_sensor but I am not following there the state names is
<img width="1032" alt="Screenshot 2021-12-10 at 19 22 58" src="https://user-images.githubusercontent.com/59267060/145630413-7651f54b-1d79-4c16-baac-b61dd64771fb.png">

**Comment by fuatakgun** [comment, marked as answer]
Should be fixed with latest configuration option
